packages/db/prisma/schema.prisma

```diff
@@ -11,6 +11,11 @@ datasource db {
   directUrl = env("DIRECT_URL")
 }
 
+model User {
+  id       String    @id
```

Comment:
using a string for id because Clerk IDs are strings, for example `user_2XRbvOxY8SRnRw5oGFlXcYwN9J8`